pkg/providers/vsphere/virtualmachine/backup.go

```diff
@@ -198,8 +210,11 @@ func BackupVirtualMachine(opts BackupVirtualMachineOptions) error {
 
 		// Set the VirtualMachine's backup version annotation once reconfigure succeeds.
 		if pkgcfg.FromContext(opts.VMCtx).Features.VMIncrementalRestore {
-			setLastBackupVersionAnnotation(opts.VMCtx.VM, opts.BackupVersion)
-			// TODO: add a condition to indicate backup has resumed
+			setBackupVersionAnnotation(opts.VMCtx.VM, opts.BackupVersion)
+			conditions.MarkTrue(
```

Comment:
Is the expectation of this true condition to report the time of the last backup? Because the last transition time will only be updated when there is an actual transition.

Reply:
Thanks for the tip, BV. I'm updating the condition's message with the latest version now. This should also force a lastTransition time update.